Add Idasen Desk connection buttons (#29546)
home-assistant/home-assistant.github.io: source/_integrations/idasen_desk.markdown

<div class='note'>
If you see an "No unconfigured devices found" message, make sure that the desk is in Bluetooth pairing mode. For that, press the small button with the Bluetooth logo on the controller until it starts blinking (about 3 seconds).
</div>

## Connect/Disconnect buttons

> This integration provides two buttons to connect and disconnect to/from the desk using Bluetooth. These can be used to automate connecting to the desk only when needed, for example, in order to avoid keeping a constant connection, if connection count is limited, for example.

Reviewer: ```suggestion
This integration provides two buttons to connect and disconnect to/from the desk using Bluetooth. These can be used to automate connecting to the desk only when needed, for example, to avoid keeping a constant connection when the available connection slots are limited.
```
